frontend/src/components/DocumentViewer/DocumentViewBoxNav.tsx

```diff
@@ -1,29 +1,28 @@
 import ThemedButton from '@src/components/ThemedButtons/ThemedButton';
 
-import './DocumentViewBoxHeader.css';
+import './DocumentViewBoxNav.css';
 
-interface DocumentViewBoxHeaderProps {
+interface DocumentViewBoxNavProps {
 	documentIndex: number;
 	documentName: string;
 	numberOfDocuments: number;
 	onPrevious: () => void;
 	onNext: () => void;
 }
 
-function DocumentViewBoxHeader({
+function DocumentViewBoxNav({
 	documentIndex,
 	documentName,
 	numberOfDocuments,
 	onPrevious,
 	onNext,
-}: DocumentViewBoxHeaderProps) {
+}: DocumentViewBoxNavProps) {
 	const documentNumber = `${documentIndex + 1} out of ${numberOfDocuments}`;
 
 	return (
-		<div className="view-documents-header">
-			<h2>View documents</h2>
+		<div className="view-documents-nav">
 			<div className="nav">
```

Comment:
Turns out you don't need this extra div, you can simply have your nav directly within the "view-documents-nav".